lang/cs/Org.Apache.REEF.Common/Avro/AvroHttpSerializer.cs

```diff
@@ -39,24 +39,24 @@ public static AvroHttpRequest FromBytes(byte[] serializedBytes)
         }
 
         /// <summary>
-        /// Conver bytes which contains Json string into AvroHttpRequest object
+        /// Convert bytes which contains Json string into AvroHttpRequest object
         /// </summary>
         /// <param name="serializedBytes">The serialized bytes.</param>
         /// <returns>AvroHttpRequest.</returns>
-        public static AvroHttpRequest FromBytesWithJoson(byte[] serializedBytes)
```

Comment:
Public method in non-example non-test code => needs to go through "Obsolete in 0.14" path